src/dmd/chkscanf.d

```diff
@@ -0,0 +1,641 @@
+/**
+ * Check the arguments to `scanf` against the `format` string.
+ *
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * Copyright:   Copyright (C) 1999-2020 by The D Language Foundation, All Rights Reserved
+ * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
+ * License:     $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/chkscanf.d, _chkscanf.d)
+ * Documentation:  https://dlang.org/phobos/dmd_chkscanf.html
+ * Coverage:    https://codecov.io/gh/dlang/dmd/src/master/src/dmd/chkscanf.d
+ */
+
+module dmd.chkscanf;
+
+import core.stdc.stdio : scanf;
+
+import dmd.errors;
+import dmd.expression;
+import dmd.globals;
+import dmd.mtype;
+import dmd.target;
+
+import std.ascii : isDigit;
```

Comment:
We do not use the standard library in the compiler to avoid the chicken and egg problem. You will most likely find a similar function in either the root directory or most definitely in the lexer class.

Reply:
Besides, the implementation is *so trivial* it just isn't worth the dependency.

Reply:
Fixed.